test/no-process-exit.js

```diff
@@ -29,6 +32,9 @@ ruleTester.run('no-process-exit', rule, {
 		'process.once("SIGINT", () => { process.exit(1); })',
 		'process.once("SIGINT", () => process.exit(1))',
 		'process.once("SIGINT", () => { if (true) { process.exit(1); } })',
+		'const {workerData, parentPort} = require(\'worker_threads\')\n\nprocess.exit(1);',
+		'import {workerData, parentPort} from \'worker_threads\'\n\nprocess.exit(1);',
+		'import foo from \'worker_threads\'\n\nprocess.exit(1);',
 		''
```

Comment:
this empty code is introduced in #1 , but seems useleess

Reply:
I thought it was meant just to make sure the rule doesn't trigger by nothing. I just left it as is but I can remove it if needed.

Reply:
Done

Reply:
Actually, I'm not sure about it, maybe @sindresorhus need review this.

let's keep it unresolved

Reply:
Sure

Reply:
Makes sense to me to keep it. Just to be safe. We should really add it to all the rules.

Reply:
Ok. I'll restore it later today. Should I do it for others here or in a separate PR?

Reply:
Oh it got merged

Reply:
Doesn't matter. We should add it to all the rules.